components/Layout.js

```diff
@@ -25,7 +26,11 @@ export default function Layout(props) {
 
   return (
     <>
-      <MetaData language={props.locale} data={props.meta}></MetaData>
+      <MetaData
+        language={props.locale}
+        data={props.meta}
+        lang={!props.locale ? 'en' : props.locale}
```

Comment:
I think `lang` prop can be removed since its not being used in <MetaData> and that `language` is doing the same thing.